### ISSUE TITLE
Jenkins build fails due to out-of-date WordNet resource file

### ISSUE THREAD

**Issue by logological** [dkpro/dkpro-similarity#44]
The DKPro Similarity Jenkins build has become unstable due to a
test failing with the following error:

````
Error creating bean with name 'wordnet-en' defined in URL
[file:/srv/dkpro/de.tudarmstadt.ukp.dkpro.lexsemresource.core.ResourceFactory/resources.xml]:
Instantiation of bean failed; nested exception is
org.springframework.beans.BeanInstantiationException: Could not
instantiate bean class
[de.tudarmstadt.ukp.dkpro.lexsemresource.wordnet.WordNetResource]:
Constructor threw exception; nested exception is
java.util.MissingResourceException: Can't find bundle for base name
net.sf.extjwnl.dictionary.messages_static, locale de_DE
````

This error is caused when a JWNL properties file is used with extJWNL.
The file
`/srv/dkpro/de.tudarmstadt.ukp.dkpro.lexsemresource.core.ResourceFactory/resources.xml`
on the Jenkins server references such a file.  I don't have access to the machine, though, so I can't replace the file myself.

**Comment by reckart** [comment]
Problem is if we replace that file, we might have other tests failing (we could try though).

Solution that departs from the need to use a shared external directory during testing would be appreciated.

**Comment by logological** [comment]
Well, we don't need to replace the file.  We could keep the existing JWNL properties file, and add a new extJWNL properties file with another name.  We could then update the DKPro LSR resources.xml file to use the new extJWNL properties file rather than the old JWNL properties file.  This will work as long as nothing else on the server needs to use DKPro LSR 0.8.0 or earlier.  (If something else does need to use DKPro LSR 0.8.0 or earlier, then we can simply add a new entry to the resources.xml file pointing to the new extJWNL properties file, and then update the DKPro Similarity JUnit test to use that new entry.)

**Comment by logological** [comment]
Fixed by Commit 8581f30ae143a822afa125cd97bd9e91f3cb70ee.
